modules/virt-adding-pci-device-hyperconverged-operator.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-adding-pci-device-hyperconverged-operator_{context}"]
+= Adding PCI devices to the HyperConverged Operator
+To enable the availability of PCI devices in the cluster, add details about the PCI devices to the HyperConverged Operator in the YAML file. The PCI devices are exposed and permitted to be used in the cluster.
+
+.Procedure
+. Use the following command to add the PCI device information to the YAML file.
++
+[source,terminal]
+$ oc get hyperconverged kubevirt-hyperconverged -n openshift-cnv -o yaml
```

Comment:
This renders correctly, but as in another location I think you should still add the `----` on the top and bottom of the command

Reply:
Updated the content.